package.json

```diff
@@ -23,7 +23,7 @@
   },
   "dependencies": {
     "bindings": "1.2.1",
-    "nan": "2.0.7"
+    "nan": "^2.0.7"
```

Comment:
I have a different perspective. Suppose the nan folks find and fix a bug, and release 2.3.3. By locking to 2.3.2 that means that you'd have to modify this file again (and cut another release) just to pickup that bugfix. I guess it depends on how well nan follows semver. Since it's part of core node.js now I have good faith that they'll follow semver fairly well.

Hmm... I guess it's a trade-off between being conservative versus being more trusting, so it's a preference and I can't speak _against_ locking the version. Locking the version requires more attention and effort (to make these small changes to package.json) so my preference is to be more trusting of nan. After all, the plan is to release a new major version of node.js twice a year (which likely involves a new version of nan as well).